docs/meshstack.aws.landing-zone-tutorial.md

```diff
@@ -0,0 +1,266 @@
+---
+id: meshstack.aws.landing-zone-tutorial
+title: "Tutorial: Landing Zone"
+---
+
+> This tutorial is work in progress!
+
+Purpose: show you how to create secure landing zones for AWS in a simple step by step guide.
+
+Terms:
+
+- **Managed Account** An AWS Account managed under this landing zone.
+- **CFN** shorthand for AWS CloudFormation
+
+## Create the Organization
+
+Setup the Organization on the root account
+
+- select the "Root" Organizational Unit that's automatically created ("Organize Accounts" section) an enable "Service control policies"
+  Policy type
+- attach the following policy to the root
+  - FullAWSAccess (default policy supplied by AWS)
+  - meshstack-aws-root: Root Policy for meshstack-managed AWS organizations
+
+This policy "jails" member accounts in the organization
+
+- deny access to organizations API
+- deny access to CRUD IAM IdPs
+- todo: figure out how/if we can restrict modifications to the IAM OrganizationAccountAccessRole and to the user's own role
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "DenyAllOrganizationsApis",
+            "Effect": "Deny",
+            "Action": [
+                "organizations:*"
+            ],
+            "Resource": [
+                "*"
+            ]
+        },
+        {
+            "Sid": "DenyRestrictedIamOperations",
+            "Effect": "Deny",
+            "Action": [
+                "iam:AddClientIDToOpenIDConnectProvider",
+                "iam:CreateOpenIDConnectProvider",
+                "iam:CreateSAMLProvider",
+                "iam:DeleteOpenIDConnectProvider",
+                "iam:DeleteSAMLProvider",
+                "iam:GetOpenIDConnectProvider",
+                "iam:GetSAMLProvider",
+                "iam:ListOpenIDConnectProviders",
+                "iam:ListSAMLProviders",
+                "iam:RemoveClientIDFromOpenIDConnectProvider",
+                "iam:UpdateOpenIDConnectProviderThumbprint",
+                "iam:UpdateSAMLProvider"
+            ],
+            "Resource": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+## Automation Account
+
+Choose an automation account in which the Stack Sets will be placed. Setup these resources on the Automation Account, here expressed as CloudFormation Templates.
```

Comment:
Not sure what is meant, maybe:

Choose an automation account in which the Stack Sets will be placed. Setup these resources inside the Automation Account. Have a look at the subsequent CloudFormation Templates for a reference.